modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1334 @@
+. Provided the menu button `api` property as an *argument* which will fetch a function that will allow menu items to change the `icon` or `text` of the parent menu button.
+
+As a result, in {productname} 6.4.1, menu items can change based on the parent menu button `api` properties.
+
+=== New `highlight_on_focus` option which enables highlighting the content area on focus
+//#TINY-9277
+
+In previous versions of {productname}, the highlighted border around the editor content window did not render when the editor was `infocus`.
+
+As a consequence, this issue, whilst creating no impending impacts on the functionality of the editor, created issues for users that rely on accessibility features such as this.
+
+To fix this issue, {productname} 6.4.1, introduced some improvements to ensure that when the editor content window is `infocus`, the editor will render a visible `highlighted` border around the content window when the below option is added to the `tinymce.init()` config.
+
+.Setting `highlight_on_focus: true` option to tinymce.init().
+[source, js]
+----
+tinymce.init({
+    selector: "textarea",
+    plugins: [ 
+    ],
+    highlight_on_focus: true // This option is required to be set to `true`.
+});
+----
+
+In {productname} 6.4.1, when the ` highlight_on_focus: true` option is set, the editor will render the highlighted border for the active editor window.
+
+=== New fontsizeinput toolbar item which allows the user to set the size via input and also increase and decrease it with + and - buttons
+//#TINY-9429
+
+In previous versions of {productname}, users did not have the ability to set their font size by either direct input into a component field or by using plus `+` or minus `-` buttons.
+
+As a consequence, {productname} introduced a new UI component that gives users an option to set the default unit of measure (UOM) for font sizes via the new input field or the plus `+` or minus `-` buttons within the toolbar when the `fontsizeinput` is set within the tinymce.init() such as the below:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",  // change this value according to your HTML
+    toolbar: "fontsizeinput",
+    font_size_input_default_unit: "pt"
+});
+----
+
+[NOTE]
+More details about this component and the possible values that can be used within the new `fontsizeinput` input field can be found in `xref:available-toolbar-buttons.adoc#the-core-toolbar-buttons[fontsizeinput]`.
+
+=== Added skipFocus option to the ToggleToolbarDrawer command to preserve focus
+//#TINY-9337
+
+A focusing event that was in the initial *FloatingToolbarButton* config, in previous versions of {productname} in (2019) was causing `focus` issues with the *ToggleToolbarDrawer*. As a consequence, this meant that while the editor was been scrolled into `focus`(viewport position), and the *ToggleToolbarDrawer* command `skipFocus: boolean` was set, the toolbar draw could still take `focus` when It was not meant to.
```

Comment:
```suggestion
A focusing event that was in the initial *FloatingToolbarButton* config, in previous versions of {productname} in (2019) was causing `focus` issues with the *ToggleToolbarDrawer*. As a consequence, this meant that while the editor was scrolled into `focus`(viewport position), and the *ToggleToolbarDrawer* command `skipFocus: boolean` was set, the toolbar draw could still take `focus` when it was not meant to.
```

Reply:
@tiny-ben-tran moved your suggestion copy changes to here so i can merge.

Notes:
============
[tiny-ben-tran]
Also, for and the *ToggleToolbarDrawer* command skipFocus: boolean was set, the toolbar draw could still take focus when it was not meant to., does it matter whether skipFocus set to false or true?

@[kemister85]
@tiny-ben-tran does not seem to have any affect if the skipFocus set to false or true as it always returns true